### PR TITLE
Adds New Relic to Join Server

### PR DESCRIPTION
**Pull request by StetsenTech**
Task:
https://app.asana.com/0/1130270710807516/1137743756490022/f

Summary:
- Adds New Relic Gem for the Join Server

### CODE REVIEW

**Comment by hcli-work** [comment]
wow. that was it?

**Comment by StetsenTech** [comment]
It's the first part. There is a `yml` file I need to upload given by heroku. I think I'm going to just add it to this by making the license a env variable for the secrets file.

File: config/newrelic.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# For full documentation of agent configuration options, please refer to
+# https://docs.newrelic.com/docs/agents/ruby-agent/installation-configuration/ruby-agent-configuration
+
+common: &default_settings
+  # Required license key associated with your New Relic account.
+  license_key: <%= ENV['NEWRELIC_LICENSE'] %>
+
+  # Your application name. Renaming here affects where data displays in New
+  # Relic.  For more details, see https://docs.newrelic.com/docs/apm/new-relic-apm/maintenance/renaming-applications
+  app_name: My Application
```
Comment (sadleb) [inline config/newrelic.yml]: Can we name this Join or Join Application?

File: config/newrelic.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  # Logging level for log/newrelic_agent.log
+  log_level: info
+
+
+# Environment-specific settings are in this section.
+# RAILS_ENV or RACK_ENV (as appropriate) is used to determine the environment.
+# If your application has other named environments, configure them here.
+development:
+  <<: *default_settings
+  app_name: My Application (Development)
```
Comment (sadleb) [inline config/newrelic.yml]: Ditto, something other than My Application would be good. Just match what you use above (with the Development) specification.

File: config/newrelic.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+test:
+  <<: *default_settings
+  # It doesn't make sense to report to New Relic from automated test runs.
+  monitor_mode: false
+
+staging:
+  <<: *default_settings
+  app_name: My Application (Staging)
```
Comment (sadleb) [inline config/newrelic.yml]: Ditto on above comment.

**Pull request merged**